src/components/ModalContent/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { PureComponent } from 'react'
```

Comment:
why can't this be inside of Modal? do we even need this component?

Reply:
The summary of this PR explains `ModalContent` and why it exists, does it answer your questions?

Reply:
What I don't get is why it needs to be a separate component. You stated that it's to "designate which part of the modal is structural setup, and which part is actual content". What is the purpose of having a separate component instead of just putting all of this in Modal?

Reply:
The purpose is so that structural markup between `<Modal>` and `<ModalContent>` is treated as part of the "backdrop."  This way, if you were to, as is in the story, wrap your content in a container and maybe size up the modal content using the grid system (see "small" example) the `.container`, `.row`, and `.col` divs don't block the click handler for the "outside click" to close the modal (or rather, fire the close click callback).

Reply:
^^ This is how we support all the different types of modal sizes and layouts without building it into the modal component itself.

Reply:
That's what I thought but I didn't see any click handlers closing the modal when the backdrop is clicked